uw-research-computing/gpu-jobs.md

```diff
@@ -294,7 +294,7 @@ recreated using the attributes `Machine`, `TotalGpus`,
 
```

Comment:
Idk why I can't edit this sentence, but for the sentence above this `CUDADeviceName` and `CUDACapability` need to be updated

Reply:
@ChristinaLK see the above comment prior to merging